### DISCUSSION TITLE
Very high false positive rate.

### DISCUSSION THREAD

**Discussion by lupinthief** [rdz-oss/BattyBirdNET-Analyzer#21] [Q&A]
I'm using an Audiomoth at 384kHz and configured Batty Birdnet at 288kHz and using the UK classifier. I seem to get masses of false positive detections, particularly of common noctule, but also Daubenton's bat even when nothing visible in the spectrogram. The attached shows a cluster of detections with the mic attached (gain on the spectrogram maxed out for visualisation but low gain set on audiomoth), but I also get them without it attached, as shown in the other screengrab. 
How do I configure it not to produce so many false positives? Any other suggestions as to what is goin on? 
![image](https://github.com/user-attachments/assets/9ff088a3-fe57-4ced-816b-09779aad18ef)
![image](https://github.com/user-attachments/assets/df4a4c5d-5b30-4ca4-940a-47d9c63f537e)

Anyone have any suggestions as to what might be going on? Incidentally, the daytime BirdNet detections look pretty implausible too - much less credible than when I ran Birdnet-Pi with a different microphone.

**Comment by rdz-oss** [comment]
Hi @lupinthief ,

I have not updated the UK classifier in a while. Some background on false detections
- the classifier runs 2 times per second and checks if a bat is present and if so which one it is
- it can hence trigger falsely on the order of about 120 times per minute
- every setup can have new noise combinations unkown to the classifier as noise. The calssifier is trained to ignore 'noise' on sample data sets. If it is not contained it could be triggered
- There is a trade off between robustness to noise and sensitivity to detecting bats. The more robust to noise the more bats you will loose and vice versa

Options
- You can send me data of noise that has been classified as a bat. About 20-100 samples should be enough. i can add that to the noise set in the classifier and it should do much better
- You can try using the Bavaria classifier. Covers UK bats too and is more up to date
- You can try @pete-the-geeks South-Wales classfier 

I can send you details for uploading the noise data?

Best regards,
Richard

**Reply by rdz-oss** [reply]
On a side note, Daubenton bat call samples known to the classifier likely have the same type of background noise that are in your setup, i.e. they are the most similar via similar noise. Noctules have lower frequency calls and many in the training set are horizontal (CF) calls from flying in open space. Hence low and horizontal noises/bird calss can be confused. Can be remedied by adding the false detections to the noise training data

**Reply by rdz-oss** [reply]
Also :) if you are comfortable using Linux you can train your own classifier using a mix of data provided by BattyBirdNEt and your own. I m writing a howto on that

**Reply by lupinthief** [reply]
Thanks. I'll try Bavaria and Wales classifiers and report back. 
I'm happy to send some samples. There are plenty where the mic isn't even on, or during the day etc. I'd like to have a go at training a classifier but, not being anything like an expert on bat calls, I'm unlikely to be able to generate meaningfully accurate labels for my own recordings (other than noise, maybe).
